CHANGELOG.md

```diff
@@ -6,13 +6,96 @@
 
 The following import changes:
 
-| Old import path                                    | New import path                                            |
-| -------------------------------------------------- | ---------------------------------------------------------- |
-| `from diagrams.onprem.logging import Logstash`     | `from diagrams.elastic.elasticsearch import Logstash`      |
-| `from diagrams.onprem.monitoring import Kibana`    | `from diagrams.elastic.elasticsearch import Kibana`        |
-| `from diagrams.onprem.search import Elasticsearch` | `from diagrams.elastic.elasticsearch import Elasticsearch` |
-
-> The previous icon was the company icon, not the elasticsearch product.
+| Old import path                                                    | New import path                                               |
```

Comment:
Thank you so much. 

I have a suggestion that what do you think about split the table into 2 tables for each elastic and OCI with header titles?

```markdown
#### Elastic

.. table

#### OCI

.. table
```

Reply:
:+1: 
It will make it more easy to read!

Reply:
I'll make the change and push